GoogleMeetup/src/main/webapp/script.js

```diff
@@ -238,6 +238,34 @@ async function createHeatmap() {
         data: heatmapData
     });
     heatmap.setMap(map);
+    
+    var originArray = ['Bankstown, NSW', 'Cronulla, NSW', 'Avalon Beach, NSW'];
+    var destinationArray = ['Blacktown, NSW', 'Strathfield, NSW', 'Potts Point, NSW'];
+
+    processRequests(map);
```

Comment:
You'll need to add originArray and destinationArray here too.

An alternative is to pass in some objects with .origin and .destination properties, since that ensures the two are properly paired and doesn't rely on ordering (if one array got sorted for some reason, processRequests would not be able to handle this).

Reply:
thank you for the suggestion, that makes a lot more sense. in terms of actually implementing this feature though, there wouldn't be a destination array but rather one destination for all origins as we're focusing on routes to the midpoint. I just used the destination array for testing purposes to make sure I could render multiple routes on the map. I've fixed the code to better reflect this.